src/validations/access-checker/wearables.ts

```diff
@@ -427,38 +389,55 @@ export const wearables: Validation = {
 
     const { pointers } = deployment.entity
     const ethAddress = externalCalls.ownerAddress(deployment.auditInfo)
-
-    const resolvedPointers: SupportedAssets[] = []
-    // deduplicate pointer resolution
+    
+    const parsedPointers: SupportedAsset[] = []
     for (const pointer of pointers) {
-      const parsed = await parseUrnNoFail(pointer)
-      if (!parsed)
-        return validationFailed(
-          `Wearable pointers should be a urn, for example (urn:decentraland:{protocol}:collections-v2:{contract(0x[a-fA-F0-9]+)}:{name}). Invalid pointer: (${pointer})`
-        )
-
-      if (!alreadySeen(resolvedPointers, parsed)) resolvedPointers.push(parsed)
+      try {
+        const parsedPointer = await parseUrn(pointer)
+        if (parsedPointer === null) {
+          return validationFailed('POR FAVOR TESTEAME!!!')
+        }
+        if (parsedPointer?.type !== 'blockchain-collection-v1-asset' && 
+            parsedPointer?.type !== 'blockchain-collection-v2-asset' &&
+            parsedPointer?.type !== 'off-chain' &&
+            parsedPointer?.type !== 'blockchain-collection-third-party') {
+          return validationFailed('POR FAVOR TESTEAME 2!!!')
+        }
+        parsedPointers.push(parsedPointer)
+      } catch {
+        return validationFailed(`Wearable pointers should be a urn, for example (urn:decentraland:{protocol}:collections-v2:{contract(0x[a-fA-F0-9]+)}:{name}). Invalid pointer: (${pointer})`)   
+      }
     }
 
-    if (resolvedPointers.length > 1)
-      return validationFailed(`Only one pointer is allowed when you create a Wearable. Received: ${pointers}`)
-
-    const parsed = resolvedPointers[0]
-
-    if (!validUrnTypes.includes(parsed.type)) {
-      return validationFailed(`Could not resolve the contractAddress of the urn ${parsed}`)
+    if (parsedPointers.length > 1) {
+      const ENTITY_TIMESTAMP_IS_AFTER_ADR_45 = deployment.entity.timestamp >= ADR_45_TIMESTAMP
+      if (ENTITY_TIMESTAMP_IS_AFTER_ADR_45) {
+        return validationFailed(`Only one pointer is allowed when you create a Wearable. Received: ${pointers}`)
+      } else {
+        const stringPointersWithoutURI = new Set()
+        parsedPointers.map(parsedPointer => {
+          // uri holds the urn
+          const { uri, ...pointerWithoutURI } = parsedPointer
+          return JSON.stringify(pointerWithoutURI)
```

Comment:
Why removing `resolveSameUrn`? The name of the function scoped the behavior very cleanly :sadparrot:

Reply:
It was a mere attempt to make it simpler :'-(. Anyways, refactored to use that terminology, hope it's better now.